packages/cli-kit/src/node/node-package-manager.ts

```diff
@@ -334,7 +335,7 @@ ${token.json(options)}
       args = argumentsToAddDependenciesWithPNPM(depedenciesWithVersion, options.type)
       break
   }
-  options.stdout?.write(`Executing...${args.join(' ')}`)
+  options.stdout?.write(`Executing... ${[options.packageManager, ...args].join(' ')}`)
```

Comment:
This line is not strictly related to the PR topic, but the output seems much more helpful.